docs/noise/qcvv/parallel_xeb.ipynb

```diff
@@ -91,6 +91,44 @@
     "import numpy as np"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***Note: The entire XEB workflow can be run by calling `cirq.experiments.parallel_two_qubit_xeb` and the combined single-qubit randomized benchmarking (RB) and XEB workflows can be run by calling `cirq.experiments.run_rb_and_xeb`.***"
```

Comment:
I think it would be clearer to split this to 2 subsections as follows:
```
# Parallel XEB
## XEB with library function
## XEB step-by-step details
```